### PR TITLE
Change Inclusive Terminology for NCP Operator

### PR DESCRIPTION
**Pull request by timdengyun**
This patch is to make change inclusive terminology where appropriate
1.Change offensive term enable/disable to activate/deactivate

### CODE REVIEW

File: bundle/kubernetes/manifests/configmap.yaml [hunk 1]
```diff
@@ -554,24 +554,24 @@ data:
...
+    # DEACTIVATE to turn off all limit checks. This option requires
     # relax_scale_validation set to True, l4_lb_auto_scaling set to False, and
-    # works on Policy API only. When not disabled, NCP will enforce a pool
+    # works on Policy API only. When activated, NCP will enforce a pool
     # member limit on LBS to prevent one LBS from using up all resources on
     # edge nodes.
-    # Choices: DISABLE ENABLE CRD_LB_ONLY
-    #ncp_enforced_pool_member_limit = DISABLE
+    # Choices: DEACTIVATE ACTIVATE CRD_LB_ONLY
+    #ncp_enforced_pool_member_limit = DEACTIVATE
```
Comment (seanpang-vmware) [inline bundle/kubernetes/manifests/configmap.yaml:566]: if changing the eum of this value, do we also need to change logic that will use this option?

File: bundle/openshift4/manifests/configmap.yaml [hunk 2]
```diff
@@ -548,24 +548,24 @@ data:
...
+    # DEACTIVATE to turn off all limit checks. This option requires
     # relax_scale_validation set to True, l4_lb_auto_scaling set to False, and
-    # works on Policy API only. When not disabled, NCP will enforce a pool
+    # works on Policy API only. When activated, NCP will enforce a pool
     # member limit on LBS to prevent one LBS from using up all resources on
     # edge nodes.
-    # Choices: DISABLE ENABLE CRD_LB_ONLY
-    #ncp_enforced_pool_member_limit = DISABLE
+    # Choices: DEACTIVATE ACTIVATE CRD_LB_ONLY
+    #ncp_enforced_pool_member_limit = DEACTIVATE
```
Comment (seanpang-vmware) [inline bundle/openshift4/manifests/configmap.yaml:560]: same as above

**Review by dantingl — COMMENTED** [review]
Hi Yun,

I don't think we need to change all manifest terms from operator side, they are all from NCP side and should be fixed on NCP side.

File: bundle/kubernetes/manifests/configmap.yaml (hunk 1 above)
Comment (timdengyun) [reply bundle/kubernetes/manifests/configmap.yaml:566]: Hi Sean, Thanks for comment. The logic of using this option has been changed as well on NCP side.

**Comment by timdengyun** [comment]
Thanks for advice, I will revert changes in manifest, and let them synced from ncp side once NCP 3.2.1 ready.

> Hi Yun,
> 
> I don't think we need to change all manifest terms from operator side, they are all from NCP side and should be fixed on NCP side.

**Comment by timdengyun** [comment]
> Thanks for advice, I will revert changes in manifest, and let them synced from ncp side once NCP 3.2.1 ready.
> 
> > Hi Yun,
> > I don't think we need to change all manifest terms from operator side, they are all from NCP side and should be fixed on NCP side.

The changes below  in manifest and deploy are reverted.

bundle/kubernetes/manifests/configmap.yaml
bundle/openshift4/manifests/configmap.yaml
deploy/kubernetes/configmap.yaml
deploy/openshift4/configmap.yaml
manifest/kubernetes/rhel/ncp-rhel.yaml
manifest/kubernetes/ubuntu/ncp-ubuntu.yaml
manifest/openshift4/coreos/ncp-openshift4.yaml

**Review by seanpang-vmware — APPROVED** [review]

**Review by dantingl — APPROVED** [review]

**Pull request merged**